fpgalib/docs/GHzDAC_v8.md

```diff
@@ -0,0 +1,505 @@
+    d(56)  spare[7..0]       Not readback
+
+
+## Register Readback:
+
+Registers are readback according to the above data fields and additional bytes given here. Length of readback = 70.
+
+    d(52)  build[7..0]         Build number of FPGA code.  
+    d(53)  SRAMcount[7..0]     Count of number of starts of sram
+    d(54)  SRAMcount[15..8]    reset with start=1;
+    d(55)  JcountA[7..0]       Count of number times IndexA/B called in JumpTable
+    d(56)  JcountB[7..0]         in last (one) SRAMstart
+    d(57)  serDAC[7..0]        Readback byte of serial DAC interface
+    d(58)  sermon[7..0]        ClockMux=bit0, A_IRQ=bit1, B_IRQ=bit2 
+    d(59)  clockmon[7..0]      bit0=Aclock, bit1=Bclock, bit7=noPLLlatch=LED3
+    d(60)  spare[7..0]
+    d(61)  spare[7..0]
+
+    d(62)  ackoutI2C[7..0]     Acknowledge output byte
+    d(63)  data7[7..0]         I2C output data, last data byte
+    d(64)  data6[7..0]
+    d(65)  data5[7..0]
+    d(66)  data4[7..0]
+    d(67)  data3[7..0]
+    d(68)  data2[7..0]
+    d(69)  data1[7..0]
+    d(70)  data0[7..0]         First data byte
+
+### I2C Interface:
+
+A custom I2C interface is used for the FPGA.  Register definitions are:
+
+    stopI2C[7..0]   Stop bytes for I2C.  Set bit=1 to indicate number of I2C bytes.
+          “10000000” indicates 1 byte
+          “00100000” indicates 3 bytes
+          “00000000” indicates no operation
+    readwriteI2C[7..0]  Read or write operation for each byte.  Read=1, Write=0
+    ackI2C[7..0]        Acknowledge output bits for reading of a byte (0=ack)
+    data7[7..0]         Last data byte to be sent 
+    data6[7..0]
+    data5[7..0]
+    data4[7..0]
+    data3[7..0]
+    data2[7..0]
+    data1[7..0]
+    data0[7..0]   First data byte to be sent
+
+#### Example
+
+Suppose we wish to do the following:
+
+* Write 1st byte “7” and get acknowledge bit 1
+* Read 2nd byte “4” and send acknowledge bit 0
+* Read 3rd byte “F” and send acknowledge bit 1
+
+The we would send a Register Write command as follows:
+
+    stopI2C =      00100000  Indicates 3 bytes total
+    readwriteI2C = 01100000  Write 1st byte, read 2nd and 3rd
+    ackI2C =       x0100000  Acknowledge bits for read bytes (x=don’t care)
+    data7..0 =     00000xx7  Actual byte data to send
+
+The Register Readback would then contain:
+
+    ack_out =      00100000  (note data wrapped around to left bits
+    data_out7..0 = F4700000  & bytes, write_acknowledge=0)
+
+
+## DaisyChain Connections
+
+### Daisy Up
+
+| Quartus        | Board      |
+|----------------|------------|
+| `Daisy_out[0]` | `J31(2,1)` |
+| `Daisy_out[1]` | `J31(3,6)` |
+| `Daisy_in[0]`  | `J31(5,4)` |
+| `Daisy_in[1]`  | `J31(7,8)` |
+
+### Daisy Down
+
+| Quartus        | Board      |
+|----------------|------------|
+| `Daisy_in[2]`  | `J32(2,1)` |
+| `Daisy_in[3]`  | `J32(3,6)` |
+| `Daisy_out[2]` | `J32(5,4)` |
+| `Daisy_out[3]` | `J32(7,8)` |
+
+
+In the version GHzDAC, the LVDS are now DC coupled.
```

Comment:
I wondered the same thing and checked the .doc version; it does indeed read like this. For now I've just transferred the contents of the file, including things like this that need clarification; I'm sure it would help people a lot to do some more extensive editing.